### PR TITLE
On login links, pass the current url as next

### PR DESCRIPTION
**Pull request by Empact**
This means that the logged-in user will be sent to the source
page, which is super convenient, e.g. when registering for events.

### CODE REVIEW

**Comment by Empact** [comment]
Fixes #104

File: .travis.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ install:
   - "pip install -r requirements.txt -r requirements.test.txt"
 before_script:
   - "cd client && npm install && cd .."
-  - "cd client && webpack --config webpack.prod.config.js && cd .."
+  - "cd client && ./node_modules/.bin/webpack --config webpack.prod.config.js && cd .."
```
Comment (jessykate) [inline .travis.yml]: @Empact is this line relevant to this PR?

Comment (Empact) [reply .travis.yml]: Well it's independent bug it fixes this failure: https://travis-ci.org/embassynetwork/modernomad/builds/280742205?utm_source=github_status&utm_medium=notification

Comment (Empact) [reply .travis.yml]: I'll separate it out into another PR

Comment (Empact) [reply .travis.yml]: #297

**Comment by jessykate** [comment]
Thank you!! :heart_eyes:
